### ISSUE TITLE
Tekton pipeline dashboard and extension add-on is to be deprecated in 0.9.  

### ISSUE THREAD

**Issue by stephenkinder** [kabanero-io/docs#399]
### What version of Kabanero does this apply to?

- Specify versions: 0.9+ 

### Specify what you want documented (required):
Complete the following information:

We need to indicate that Kabanero is moving towards OpenShift Pipelines completely and not extending the Tekton environment any longer.  OpenShift will be providing its own webhook dialog and Kabanero will be transitioning to leveraging a single webhook route through the eventing operator.

#### To create a topic:
- Specify a link to the section in the navigation where the topic should go:  (?)
- Specify a first draft of the topic that you want added:   need help here.

#### To update existing topics:
- Specify links to the topics that are affected:  unsure.
- Include a copy of the current text and the exact text to which it will change. For example: Change ABC to XYZ:

**Comment by jantley-ibm** [comment]
@stephenkinder Is this just a "what's new" type of alert, or are doc changes needed everywhere tekton is mentioned for 0.9? Do we have replacement instructions for using openshift pipelines?

**Comment by anneqm** [comment]
@stephenkinder @jantley-ibm @ralanlittle Did we decide if this is still going out in Kabanero 0.9 or we waiting till kabanero 0.10?

**Comment by anneqm** [comment]
@jantley-ibm latest update on this one... we are NOT going to mention anything about this in kabanero 0.9.   @stephenkinder should this work item be closed with no action?

**Comment by jantley-ibm** [comment]
@anneqm correct, there is currently no mention of this in the documentation. Per @stephenkinder last week, pending new migration doc being produced.

**Comment by anneqm** [comment]
@jantley-ibm @stephenkinder Where are we documenting deprecation?  I haven't found any other issues for this as yet.
